pkg/controllers/management/secretmigrator/clusters_test.go

```diff
@@ -0,0 +1,175 @@
+package secretmigrator
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	apimgmtv3 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	fakes2 "github.com/rancher/rancher/pkg/generated/norman/core/v1/fakes"
+	"github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3/fakes"
+	rketypes "github.com/rancher/rke/types"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/api/core/v1"
+	apierror "k8s.io/apimachinery/pkg/api/errors"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Purely s stylistic choice but I've been naming collisions like this `metav1`

Reply:
Good point, fixed.